### PR TITLE
1917 adding expression profile

### PR DESCRIPTION
**Pull request by msevestre**

### CODE REVIEW

File: src/PKSim.Core/Model/ExpressionProfile.cs [hunk 1]
```diff
@@ -0,0 +1,11 @@
+﻿using OSPSuite.Core.Domain;
+
+namespace PKSim.Core.Model
+{
+   public class ExpressionProfile : PKSimBuildingBlock
+   {
+      public ExpressionProfile():base(PKSimBuildingBlockType.ExpressionProfile)
```
Comment (msevestre) [inline src/PKSim.Core/Model/ExpressionProfile.cs:7]: @Yuri05 This is the default building block implementation for now. Behavior to come next :)

File: src/PKSim.Infrastructure/Serialization/ORM/Mapping/ExpressionProfileMetaDataMapping.cs [hunk 2]
```diff
@@ -0,0 +1,14 @@
+﻿using FluentNHibernate.Mapping;
+using PKSim.Infrastructure.Serialization.ORM.MetaData;
+
+namespace PKSim.Infrastructure.Serialization.ORM.Mapping
+{
+   public class ExpressionProfileMetaDataMapping : SubclassMap<ExpressionProfileMetaData>
+   {
+      public ExpressionProfileMetaDataMapping()
+      {
+         Table("EXPRESSION_PROFILES");
```
Comment (msevestre) [inline src/PKSim.Infrastructure/Serialization/ORM/Mapping/ExpressionProfileMetaDataMapping.cs:10]: Expression profiles will be saved in a new table

File: src/PKSim.Infrastructure/Serialization/Xml/Serializers/ExpressionProfileXmlSerializer.cs [hunk 3]
```diff
@@ -0,0 +1,9 @@
+﻿using PKSim.Core.Model;
+
+namespace PKSim.Infrastructure.Serialization.Xml.Serializers
+{
+   public class ExpressionProfileXmlSerializer : BuildingBlockXmlSerializer<ExpressionProfile>
```
Comment (msevestre) [inline src/PKSim.Infrastructure/Serialization/Xml/Serializers/ExpressionProfileXmlSerializer.cs:5]: Nothing to serialzie specifically for now

**Review by msevestre — COMMENTED** [review]
This is implementing the default behavior for a new building block management. There is quite some stuff to do before we can have the create and edit
With the state of the implementation, we can create an empty building block, edit rename, delete etc..

**Comment by msevestre** [comment]
image
image
image

**Comment by msevestre** [comment]
@abdelr @georgeDaskalakis These are all the steps required to create a new building block. Similar steps will be required in MoBi. Start from the Explorer, nodes=>actions=>command etc..

**Review by Yuri05 — APPROVED** [review]

**Pull request merged**